src/Orchard.DisplayManagement/Theming/ThemeManager.cs

```diff
@@ -47,7 +47,7 @@ public class ThemeManager : IThemeManager
                 foreach (var theme in themeResults)
                 {
                     var t = _extensionManager.GetExtension(theme.ThemeName);
-                    if (t.Exists)
+                    if (t?.Exists ?? false)
```

Comment:
Ok. So here it was to prevent from an exception when the current selected theme was no more present on the disk. So, ok if `GetExtension()` returns a non null object. Will revert it this night asap.